docs/meshstack.azure-developer-platform-quickstart-guide.md

```diff
@@ -0,0 +1,240 @@
+
+Make sure you save the `config_tf` output and save it as a file called `config.tf`. We will use it later.
+
+### Creating the GitHub Actions Connector Building Block Definition
+
+We will again use the [meshStack Hub](https://hub.meshcloud.io) to create the new building block definition.
+
+1. Open the Github Actions Integration with AKS definition in the meshStack hub [here](https://hub.meshcloud.io/definitions/aks-github-connector).
+2. Click on “Add to meshStack” and enter your meshStack URL (if not given already).
+3. Continue the flow in meshStack.
+    1. Right away you will be asked for a “Type”. Pick “Tenant Building Block” here and select “AKS” as the supported platform.
+    2. Continue until you reach the “Dependencies” page. On this page, select the “Github Repository Creation” building block definition that you created before.
+    3. Continue until you reach the “Inputs” page. On this page you will be forced to fill in the following:
+        1. `namespace` - This refers to the AKS namespace. Open this up and set the “Source” to “Platform Tenant ID”.
+        2. `github_repo` - This refers to the GitHub repository to connect to. Open this up and set the “Source” to “Parent Building Block Output” and set the corresponding output to “repo_name”.
+        3. `GITHUB_APP_ID` - This is the value saved under “Preparation” step 1b from the previous building block definition
+        4. `GITHUB_APP_INSTALLATION_ID` - This is the value saved under “Preparation” step 2b from the previous building block definition
+        5. `GITHUB_APP_PEM_FILE` - This is the value saved under “Preparation” step 3a from the previous building block definition. Take the entire content of the file and paste it in here.
+        6. Create a new input called `config.tf` and set the Source to “Static” and the Type to “File”. Mark the input as encrypted and upload the config.tf file from the preparation step.
+4. Finish the flow and create the building block definition.
+
+🎉 The GitHub Actions Connector Building Block Definition is ready!
+
+## 3. Starter Kit
+
+The last piece of the puzzle in the guide is the starter kit that will bundle up all previously created resources into one, easy-to-use package for application teams.
+
+### Preparation
+
+We will create resources within meshStack. This means we need an API Key to interact with meshStack in our Terraform code.
+
+- Create a new API Key in meshStack in the Admin Area. Make sure you save the **API key ID** & **API key secret** somewhere, as you’ll need it later.
+- Give it the following permissions:
+  - “Admin: Create and Update Building Blocks in any Workspace”
+  - “Admin: List Building Blocks in any Workspace”
+  - “Admin: Create and Update Projects in any Workspace”
+  - “Admin: List Projects in any Workspace”
+  - “Admin: Create and update Tenants in any Workspace as well as import existing platform tenants”
+  - “Admin: List Tenants in any Workspace”
+
+### Creating the Building Block Definition
```

Comment:
as there are quite a lot definitions here I think giving this one a name as well will help setting the puzzle pieces together